### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by nithyatsu**

### CODE REVIEW

File: docs/content/reference/resource-schema/core-schema/gateway/index.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ weight: 401
 | name | y | The name of your Gateway. | `'gateway'`
 | location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
 | [properties](#properties) | y | Properties of the resource. | [See below](#properties)
+| [tls](#TLS) | n | TLS Configurations of the gateway. | [See below](#properties)
+
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/gateway/index.md:21]:
```suggestion
```

File: docs/content/reference/resource-schema/core-schema/gateway/index.md [hunk 2]
```diff
@@ -26,6 +28,15 @@ weight: 401
 | [hostname](#hostname) | n | The hostname information for this gateway. | [See below](#hostname)
 | [routes](#routes) | y | The routes attached to this gateway. | [See below](#routes)
 
+### TLS
+
+| Key  | Required | Description | Example |
+|------|:--------:|-------------|---------|
+| sslPassThrough| y | Determines if the gateway passthrough the TLS traffic to backend server. Currently only `true` is supported. | `true`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/gateway/index.md:34]:
```suggestion
| sslPassthrough | n | Configures the gateway to passthrough encrypted SSL traffic to the target container. Disables route-based matching and routing. Defaults to 'false'. | `true`
```

Comment (AaronCrawfis) [reply docs/content/reference/resource-schema/core-schema/gateway/index.md:34]: Doesn't this value default to "false" if omitted?

File: docs/content/reference/resource-schema/core-schema/gateway/index.md [hunk 3]
```diff
@@ -26,6 +28,15 @@ weight: 401
...
+
+With sslPassThrough set to `true`, the gateway can only support SNI routing. Path based routing cannot be supported.
+
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/gateway/index.md:37]:
```suggestion
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: docs/content/reference/resource-schema/core-schema/gateway/index.md (hunk 2 above)
Comment (nithyatsu) [reply docs/content/reference/resource-schema/core-schema/gateway/index.md:34]: it defaults to true if omitted...

File: docs/content/reference/resource-schema/core-schema/gateway/index.md [hunk 4]
```diff
@@ -17,6 +17,7 @@ weight: 401
 | name | y | The name of your Gateway. | `'gateway'`
 | location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
 | [properties](#properties) | y | Properties of the resource. | [See below](#properties)
+| [tls](#TLS) | n | TLS Configurations of the gateway. | [See below](#properties)
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/gateway/index.md:20]: This should go under the `Properties` section, as it's nested under properties and not inline with it

File: docs/content/reference/resource-schema/core-schema/gateway/index.md [hunk 5]
```diff
@@ -26,6 +27,14 @@ weight: 401
...
 | [routes](#routes) | y | The routes attached to this gateway. | [See below](#routes)
 
+### TLS
+
+| Key  | Required | Description | Example |
+|------|:--------:|-------------|---------|
+| sslPassThrough| y | Determines if the gateway passthrough the TLS traffic to backend server. Currently only `true` is supported. | `true`
+
+With sslPassThrough set to `true`, the gateway can only support SNI routing. Path based routing cannot be supported.
+
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/gateway/index.md:37]:
```suggestion
```

Let's incorporate this into the Description of `sslPassthough`. Added a suggestion, but feel free to update with additional context if needed.

**Pull request closed without merging**